### PR TITLE
feat(crux): add vaultwarden template

### PR DESCRIPTION
**Pull request by polaroi8d**
Add the Vaultwarden template to the list.

### CODE REVIEW

**Review by m8vago — APPROVED** [review]

File: web/crux/templates/vaultwarden.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "internal": 80
+          }
+        ],
+        "volumes": [
+          {
+            "name": "data",
+            "path": "/data"
+          }
+        ],
+        "environment": [],
```
Comment (nandor-magyar) [inline web/crux/templates/vaultwarden.json:33]: We need `DOMAIN` environment variable, containing the site URL: https://vault.example.com/.

File: web/crux/templates/vaultwarden.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "images": [
+    {
+      "name": "vaultwarden-server",
+      "registryName": "Vaultwarden / Docker HUB",
+      "image": "server",
+      "tag": "latest",
+      "config": {
+        "name": "vaultwarden",
+        "ports": [
```
Comment (nandor-magyar) [inline web/crux/templates/vaultwarden.json:21]: Websocket is also an option for that port 3012 is used, we might not be able to deploy that correctly with crane's current ingress controller features, without modification,
It is possible with dagent though, using custom labels. 
We need a rule like:
```
Host(`vault.dyrectorio.org`) && PathPrefix(`/notifications/hub/negotiate`)
```
That routes to port 3012, all else can go to 80.

Comment (nandor-magyar) [reply web/crux/templates/vaultwarden.json:21]: Custom labels for Traefik to do the custom routing are missing.

Comment (polaroi8d) [reply web/crux/templates/vaultwarden.json:21]: I don't know Traefik labels.

**Pull request merged**